src/Zend/Form/Element.php

```diff
@@ -2086,7 +2086,7 @@ protected function _loadFilter(array $filter)
         } else {
             $r = new ReflectionClass($name);
             if ($r->hasMethod('__construct')) {
-                $instance = $r->newInstanceArgs((array) $filter['options']);
+                $instance = $r->newInstanceArgs(array_values((array) $filter['options']));
```

Comment:
`$filter['options']` in this case had an associative array (`['bar' => 'baz']`), which PHP8 treated as named parameters due to the key `bar`. Forcing these to just the array values since this code was written before named parameters were a thing, so there was no intent to support that at the time.